.github/workflows/lint-python.yml

```diff
@@ -0,0 +1,54 @@
+name: Lint
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+concurrency:
+  group: lint-python-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  python-lint:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        python-versions: [ '3.8', '3.11' ]
+
+    name: python-lint
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+
+      - name: Set up Python ${{ matrix.python-versions }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-versions }}
+
+      - name: Install Python dependencies
+        run: pip install pylint
+
+      - name: Lint
+        uses: wearerequired/lint-action@v2
+        with:
+          pylint: true
```

Comment:
Will use Pylint to check the files with extensions py
  Linting files in /home/runner/work/nextcloud-talk-recording/nextcloud-talk-recording with Pylint …
  Pylint found 454 errors (failure)


cc @danxuliu